packages/page-accounts/src/Sidebar/Identity.tsx

```diff
@@ -90,6 +92,20 @@ function Identity ({ address, identity }: Props): React.ReactElement<Props> | nu
                 </div>
               </div>
             )}
+            {!!subs?.length && (
+              <div className='tr subs'>
+                <div className={subs.length > 1 ? 'th top' : 'th'}>{t<string>('subs')}</div>
```

Comment:
Can't remember how you go about plurals. If you cater for it or not. I'm happy to add a condition and have "sub" and "subs"

Reply:
Thinking about it, it's actually much better, committing that.